README.md

```diff
@@ -138,14 +138,26 @@ Paths provided are relative to the current directory from which you execute the
 
 There are plenty of examples you can use to start off:
 
-- `example`: contains an example app with version 2 of the specification. It will give you an idea how to annotate your comments in order to include them in the output specification.
+- `examples/js`: contains an example app with version 2 of the specification. It will give you an idea how to annotate your comments in order to include them in the output specification.
+- `examples/ts`: contains an example TypeScript app with version 2 of the specification. 
 - `test/cli`: CLI tests you can read to get ideas about the available functionalities of the CLI. (apart from the obvious help menu)
 - `test/example`: various assets for tests you can also re-use for starting definitions, routes, etc.
 
 ### CLI
 
 On top of the Node API, there is also a [command line interface](./docs/CLI.md).
 
+
+### Using with TypeScript at Runtime
+
+Yes, you can use swagger-jsdoc with your TypeScript project. There are some adjustments in the configuration necessary, though.
+1. You can install types via `@types/swagger-jsdoc`.
```

Comment:
This is not required, take it out of the list. Please move to a new paragraph as a suggestion with 1 sentence of the benefit of using the package.